### PR TITLE
Rename universe functionality to accomodate transfer proofs.

### PR DESCRIPTION
**Pull request by ffranr**
Our universe functionality now handles transfer proofs in addition to issuance proofs. This PR renames functions and structures for clarity and correctness.

### CODE REVIEW

**Comment by dstadulis** [comment]
Just code clean up,
we're using transfer proofs through this code areas, so this increases the accuracy

File: universe/auto_syncer.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ type FederationPushReq struct {
 // FederationIssuanceBatchPushReq is used to push out a batch of new issuance
 // events to all or some members of the federation.
 type FederationIssuanceBatchPushReq struct {
-	IssuanceBatch []*IssuanceItem
+	IssuanceBatch []*Item
```
Comment (guggero) [inline universe/auto_syncer.go:79]: Do we want to rename this (and the struct name) as well? Or is this actually just for issuances?

**Review by guggero — APPROVED** [review]
LGTM :tada:

File: universe/auto_syncer.go (hunk 1 above)
Comment (ffranr) [reply universe/auto_syncer.go]: Yes, I've added more commits to address this.

**Comment by ffranr** [comment]
There are probably more changes that we need to make along the same lines, but I would prefer to handle those in another PR unless they're pressing.

**Review by guggero — APPROVED** [review]

**Review by jharveyb — APPROVED** [review]
LGTM 👍🏽 

Will definitely be helpful when debugging these areas in the future.

**Pull request merged**